src/nodeViews/RTENode.ts

```diff
@@ -0,0 +1,208 @@
+import { exampleSetup } from "prosemirror-example-setup";
+import { redo, undo } from "prosemirror-history";
+import { keymap } from "prosemirror-keymap";
+import type { Node, Schema } from "prosemirror-model";
+import type { Transaction } from "prosemirror-state";
+import { EditorState } from "prosemirror-state";
+import { Mapping, StepMap } from "prosemirror-transform";
+import type { Decoration } from "prosemirror-view";
+import { DecorationSet, EditorView } from "prosemirror-view";
+
+/**
+ * A NodeView (https://prosemirror.net/docs/ref/#view.NodeView) that represents a
+ * nested rich text editor interface.
+ */
+export class RTENodeView<LocalSchema extends Schema> {
+  // The parent DOM element for this view. Public
+  // so it can be mounted by consuming elements.
+  public dom = document.createElement("div");
```

Comment:
Is the 'dom' name a Prosemirror convention we're borrowing? Something like 'root' might be clearer otherwise.

Reply:
Yes, I'll make that clearer – how's `nodeViewElement`?